style.css

```diff
@@ -349,7 +367,7 @@ border-radius: 8px;
   width: 200px;
   height: 40px;
   list-style: none;
-  margin: -10px 0px 0px -150px;
+  margin: -10px 0 0 -150px;
 }
 
 .Button {
```

Comment:
- [ ] Nice try on your buttons. The design has provided you with some nice properties for your button interactions. Kindly implement them to further make your design more appealing and professional.
Here's a sample of the button pallet from the project requirement :point_down: 

![image](https://user-images.githubusercontent.com/99715640/219016825-ba361a51-a56c-48bc-9398-6d3221e77fc0.png)

Reply:
the buttons are okay it's just that you are using a different browser.